tasks/main.yml

```diff
@@ -20,6 +20,16 @@
   timezone:
     name: "{{ ntp_timezone }}"
 
+- name: Populate service facts
+  service_facts:
+
+- name: Disable systemd-timesyncd
```

Comment:
This would need to be `Disable systemd-timesyncd if it's running but ntp is enabled.`

Reply:
updated